### PR TITLE
Add Mongolian locale relativeTime

### PR DESCRIPTION
**Pull request by munkhorgil**

### CODE REVIEW

**Comment by iamkun** [comment]
Expected value to equal:
      ["M", "MM", "d", "dd", "future", "h", "hh", "m", "mm", "past", "s", "y", "yy"]

**Comment by munkhorgil** [comment]
@iamkun Can you give me a received value?

**Comment by munkhorgil** [comment]
@iamkun pass

File: src/locale/mn.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const locale = {
   months: 'Нэгдүгээр сар_Хоёрдугаар сар_Гуравдугаар сар_Дөрөвдүгээр сар_Тавдугаар сар_Зургадугаар сар_Долдугаар сар_Наймдугаар сар_Есдүгээр сар_Аравдугаар сар_Арван нэгдүгээр сар_Арван хоёрдугаар сар'.split('_'),
   weekdaysShort: 'Ням_Дав_Мяг_Лха_Пүр_Баа_Бям'.split('_'),
   monthsShort: '1 сар_2 сар_3 сар_4 сар_5 сар_6 сар_7 сар_8 сар_9 сар_10 сар_11 сар_12 сар'.split('_'),
+  weekStart: 1,
```
Comment (iamkun) [inline src/locale/mn.js]: please remove this line, according to ISO format, should start from sunday I think?

File: src/locale/mn.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const locale = {
   months: 'Нэгдүгээр сар_Хоёрдугаар сар_Гуравдугаар сар_Дөрөвдүгээр сар_Тавдугаар сар_Зургадугаар сар_Долдугаар сар_Наймдугаар сар_Есдүгээр сар_Аравдугаар сар_Арван нэгдүгээр сар_Арван хоёрдугаар сар'.split('_'),
   weekdaysShort: 'Ням_Дав_Мяг_Лха_Пүр_Баа_Бям'.split('_'),
   monthsShort: '1 сар_2 сар_3 сар_4 сар_5 сар_6 сар_7 сар_8 сар_9 сар_10 сар_11 сар_12 сар'.split('_'),
+  weekStart: 7,
```
Comment (iamkun) [inline src/locale/mn.js]: It will be great if you could remove this line.

Comment (munkhorgil) [reply src/locale/mn.js]: done

**Comment by iamkun** [comment]
Cool thanks

**Pull request merged**